docs/self-managed/setup/deploy/amazon/amazon-eks/terraform-setup.md

```diff
@@ -226,29 +226,81 @@ Users can generate access to the Amazon EKS cluster via the `AWS CLI`.
 aws eks --region <region> update-kubeconfig --name <clusterName>
 ```
 
-### Terraform AWS IAM permissions
+### Terraform AWS IAM Permissions
```

Comment:
Considering the word "permissions" is lowercase in `AWS IAM permissions,` I'd assume it remains lowercase as well. This might be something for DevEx to have a final view on to keep everything aligned.

As far as I remember, we don't follow any AP or Chicago style, but just `Uppercase` followed by `lowercase`, with proper nouns being the exception.

So `Terraform AWS IAM permissions` should be correct.